src/libmongoc/tests/unified/entity-map.c

```diff
@@ -551,6 +552,7 @@ session_opts_new (bson_t *bson, bson_error_t *error)
 
    bp = bson_parser_new ();
    bson_parser_bool_optional (bp, "causalConsistency", &causal_consistency);
+   bson_parser_bool_optional (bp, "isSnapshot", &snapshot);
```

Comment:
I was confused by this change not being accompanied by a change in the supported schemas, but I forgot that the unified test format doesn't specify these options but rather accepts anything that the `startSession` helper supports. Adding this for the benefit of other reviewers.

Reply:
Ah yep, I don't think there were any unified test format changes here.